### ISSUE TITLE
[Question]: How about multi-lingual?

### ISSUE THREAD

**Issue by sensoryfox** [infiniflow/ragflow#837]
### Describe your problem

Hey, everybody!
I'm sure you are doing a very cool thing, rolled out at my place, but faced with the fact that in my language (Russian), nothing works. I had to fix everything by hand, including the work of regular expressions. 

Up to this point, I was getting something like this from the bot: "нучто-то,ядумаю,вотздесьвотестькнопочкаструбкой,что-то,наверное,надозаказатьпоповодутого,заказатьобратныйзвонок."

I wanted to look in the direction of expanding the possibilities of language management and put this task in a separate branch. Do you have any plans to enable multi-language for ragflow?

**Comment by KevinHuSh** [comment]
Yes, of course. But we're lack of native speeker for other languages which leads to that we can't evaluate our work or solutions.
Welcome to contribute, it's a international community isn't it.
